### PR TITLE
Update base_model guidance to reflect that base_model is an array now.

### PR DESCRIPTION
**Pull request by davanstrien**
I have yet to make any changes to the metadata UI screenshots. cc @julien-c @osanseviero

### CODE REVIEW

**Comment by davanstrien** [comment]
Switched examples back to single model ID. We can also maybe keep the UI screenshots as they are and focus on the simple case?

File: docs/hub/model-cards.md [hunk 1]
```diff
@@ -104,10 +104,10 @@ If it's not specified, the Hub will try to automatically detect the library type
 
 ### Specifying a base model
 
-If your model is a fine-tune or adapter of a base model, you can specify the base model in the model card metadata section:
+If your model is a fine-tune or adapter of a base model, you can specify the base model in the model card metadata section. This information can also be used to indicate if your model is a merge of multiple existing models. The `base_model` field can either be a single model ID or an array of one or more base_models (specified by their Hub identifier). 
 
 ```yaml
-base_model: HuggingFaceH4/zephyr-7b-beta
+base_model:  HuggingFaceH4/zephyr-7b-beta
```
Comment (pcuenca) [inline docs/hub/model-cards.md:110]:
```suggestion
base_model: HuggingFaceH4/zephyr-7b-beta
```

(nit, leading space)

File: docs/hub/model-cards.md [hunk 2]
```diff
@@ -104,10 +104,10 @@ If it's not specified, the Hub will try to automatically detect the library type
 
 ### Specifying a base model
 
-If your model is a fine-tune or adapter of a base model, you can specify the base model in the model card metadata section:
+If your model is a fine-tune or adapter of a base model, you can specify the base model in the model card metadata section. This information can also be used to indicate if your model is a merge of multiple existing models. The `base_model` field can either be a single model ID or an array of one or more base_models (specified by their Hub identifier).
```
Comment (pcuenca) [inline docs/hub/model-cards.md:107]:
```suggestion
If your model is a fine-tune or adapter of a base model, you can specify the base model in the model card metadata section. This information can also be used to indicate if your model is a merge of multiple existing models. The `base_model` field can either be a single model ID, or a list of one or more base_models (specified by their Hub identifier). 
```

just an idea, use what sounds better to you :)

**Review by julien-c — APPROVED** [review]
keeping the existing screenshots sound good to me

**Comment by davanstrien** [comment]
Thanks for the reviews @julien-c @pcuenca :)

**Pull request merged**